### ISSUE TITLE
Pagine eventi elettorali

### ISSUE THREAD

**Issue by iltempe** [HackForItaly/politicamentecorretto#15]
ho creato due pagine per gli eventi elettorali del 2018. Potenzialmente potremmo farne infinite. Forse è il caso di mettere più label nel repository delle segnalazioni che indichino la promessa a quale evento è legata?

**Comment by cristigalas** [comment]
@iltempe concordo. Dovremmo aggiungere almeno due label: Politiche 2018 e Regionali 2018. 
Intanto ho aggiunto 4 label tematiche: ambiente, welfare, fiscalità, pensioni con cui catalogare le promesse elettorali.

**Comment by cristigalas** [comment]
@iltempe il titolo del menù “Elezioni” non mi convince ma pensiamoci... intanto però modificherei il testo di spiegazione delle due pagine. Secondo me sarebbe meglio scrivere: In questa pagina raccoglieremo le promesse elettorali delle elezioni politiche 2018 (e delle elezioni regionali 2018). 
Dire “racconteremo le elezioni” non è chiaro, non rende l’idea a mio avviso...
